bin/p2-label/main.go

```diff
@@ -65,6 +66,11 @@ func main() {
 		fmt.Printf("%s/%s: %s\n", labelType, *showID, labelsForEntity.Labels.String())
 		return
 	case CmdApply:
+		if *applySubjectSelector == "" && *applySubjectID == "" {
+			fmt.Fprint(os.Stderr, "Must pass either an ID or a selector for objects to apply the given label to")
+			exitCode = 1
+			break
+		}
```

Comment:
should you check and err if they're both set? looks like the code will use the selector and not the ID in that case which could be surprising